### ISSUE TITLE
shuffling songs and won't play

### ISSUE THREAD

**Issue by boris19** [music-assistant/hass-music-assistant#2834]
### What version of Music Assistant has the issue?

2.3.0b6

### What version of the Home Assistant Integration have you got installed?

2024.8.0

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

the songs start to change quickly and the sound does not play. please watch the video. what problem could it be?

https://github.com/user-attachments/assets/f27fcac9-d20a-40e0-8675-975fad8e8a98

### How to reproduce

I just opened Music Assistant in the menu.

### Music Providers

Spotify

### Player Providers

Snapcast

### Full log output

[log.log](https://github.com/user-attachments/files/16803242/log.log)

### What version of Home Assistant Core are your running

2024.8.2

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Windows

**Comment by OzGav** [comment]
@SantiagoSotoC some major errors in the log. Is this Snapcast or in core?

**Comment by SantiagoSotoC** [comment]
It seems a dependency error is rare, it is as if the dependency is not installed or some network error

**Comment by SantiagoSotoC** [comment]
@boris19 should try the  beta 9 Marcel made changes, my idea was to try them but my day was more complicated than expected

**Comment by boris19** [comment]
yes, I tried to update to beta11, but it writes a lot of environment errors in the logs. see log.

[upeatelog.log](https://github.com/user-attachments/files/16811666/upeatelog.log)

**Comment by boris19** [comment]
In the shuffle issue, i disabled shuffle mode. I will test if issue come back. i suspect the error was caused by music shuffle being turned on.

**Comment by OzGav** [comment]
Beta 12 is building. Try again then. Sorry for the inconvenience

**Comment by marcelveldt** [comment]
Please ignore versions b9-b15 people as those were flawed after our base container switch. 
Update to b16 and check if you still have any issues (I don't think so)

**Comment by OzGav** [comment]
@boris19 please confirm this has been fixed

**Comment by boris19** [comment]
I saw error today too. Shuffle is disabled. 

https://github.com/user-attachments/assets/49c18236-cbb7-4f23-b1ff-0515b22b3ecf

[log2.log](https://github.com/user-attachments/files/16824752/log2.log)

**Comment by boris19** [comment]
I extended the sync time to 720 minutes and noticed that just before this happened a Spotify sync was done. Couldn't it be made to do this synchronization only at a certain defined time ? Personally I would set it at night when it is not playing.

**Comment by boris19** [comment]
I have to roll back to b8. I have to restart MA every time it happen. And afrer restart MA lost Spotify autentification. In b16 this is happening every one hour.

**Comment by OzGav** [comment]
No one else is reporting an issue with Spotify. Please check the logs for additional information. Also keep this issue report to just the original topic.

**Comment by boris19** [comment]
yes I see. I rolled back to b8, is more stable now.

**Comment by OzGav** [comment]
I don’t understand what the problem is

**Comment by boris19** [comment]
I did some tests. I changed the Snapcast codec to opus. in b8 it played without any problems just when I loaded it, that I logged in as a second user in HA, and once I played music through one user and then through the other Snapcast stopped playing. i did the same tests on b17 but i'm still testing it because today is a free day in the building. b17 behaves better, but sometimes it happens that MA plays, but Snapcast does not, in that case you just have to put another song, so press the next button and Snapcast starts playing.
